trulens_eval/trulens_eval/feedback/provider/base.py

```diff
@@ -249,15 +319,69 @@ def generate_score_and_reasons(
             return score, reasons
 
         else:
-            score = mod_generated_utils.re_0_10_rating(response) / normalize
+            score = (
+                mod_generated_utils.re_configured_rating(
+                    response,
+                    min_score_val=min_score_val,
+                    max_score_val=max_score_val
+                ) - min_score_val
+            ) / (max_score_val - min_score_val)
             warnings.warn(
                 "No supporting evidence provided. Returning score only.",
                 UserWarning
             )
             return score, {}
 
+    # TODO: to be deprecated soon
+    def context_relevance_old(
+        self,
+        question: str,
+        context: str,
+        temperature: float = 0.0,
+    ) -> float:
+        """
+        Uses chat completion model. A function that completes a template to
+        check the relevance of the context to the question.
+
+        !!! example
+
+            ```python
+            from trulens_eval.app import App
+            context = App.select_context(rag_app)
+            feedback = (
+                Feedback(provider.context_relevance_with_cot_reasons)
+                .on_input()
+                .on(context)
+                .aggregate(np.mean)
+                )
+            ```
+
+        Args:
+            question (str): A question being asked.
+
+            context (str): Context related to the question.
+
+        Returns:
+            float: A value between 0.0 (not relevant) and 1.0 (relevant).
+        """
+
+        return self.generate_score(
+            system_prompt=prompts.CONTEXT_RELEVANCE_SYSTEM_OLD,
+            user_prompt=str.format(
+                prompts.CONTEXT_RELEVANCE_USER,
+                question=question,
+                context=context
+            ),
+            temperature=temperature
+        )
+
     def context_relevance(
```

Comment:
add a full docstring here incl. a usage example

Reply:
Added docstring for now. 

I'm thinking adding the usage example after the refactoring is done (i.e. `_verb_confidence` and `_cot_reasons` as feedback params)